crates/gossip/src/config.rs

```diff
@@ -0,0 +1,49 @@
+//! Configuration parameters for the gossip module.
+
+/// Configuration parameters for K2Gossip.
+///
+/// This will be set as a default by the [K2GossipFactory](crate::K2GossipFactory).
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct K2GossipConfig {
+    /// The maximum number of bytes of op data to request in a single gossip round.
+    ///
+    /// This applies to both "new ops" which is the incremental sync of newly created data, and
+    /// to a DHT mismatch in existing data. The new ops are synced first, so the remaining capacity
+    /// is used for the mismatch sync.
+    ///
+    /// The maximum size of an op is host dependant, but assuming a 1MB limit, this would allow
```

Comment:
Easier to read with a space between value and unit.
```suggestion
    /// The maximum size of an op is host dependant, but assuming a 1 MB limit, this would allow
```

Reply:
So no?

Reply:
I applied this change in a batch locally, sorry that doesn't mark it as resolved here. I can commit as batch on the UI if that's easier. I'm just aware that won't work with conventional commits at some point so I'm trying to work locally

Reply:
Ah okay, good to know, then just "carry on" =)